### PR TITLE
Fixed issue + updated feature : usergroups view

### PR DESCRIPTION
**Pull request by papa-mb**
HI,
I've created a bug #12051 and in this ticket i proposed to update this feature to separate the groups created by the current user & the groups to which this  user belong :)
Thank's by advance

### CODE REVIEW

File: application/models/UserGroup.php [hunk 1]
```diff
@@ -247,7 +247,7 @@ public function getbuttons()
         return $button;
     }
 
-    function search()
+    function searchMine($mine)
```
Comment (olleharstedt) [inline application/models/UserGroup.php]: Always add PHPDoc annotations, please.

File: application/views/admin/usergroup/usergroups_view.php [hunk 2]
```diff
@@ -62,21 +70,103 @@
...
+                    'ajaxUpdate' => true,
+                   ));
+            ?>
+        </div>
+    </div>
+    
+    <h4><?php
+        if (!Permission::model()->hasGlobalPermission('superadmin','read'))
+        {
+            eT('Groups to which i belong');
```
Comment (olleharstedt) [inline application/views/admin/usergroup/usergroups_view.php]: Capital `I` here.

**Comment by papa-mb** [comment]
Hi 
I made the modifications :)

File: application/models/UserGroup.php [hunk 3]
```diff
@@ -246,7 +246,16 @@ public function getbuttons()
...
+     * This function search usergroups for a user
+     * If $mine = true then usergroups are those that have been created by the current user
+     * else this function provides usergroups which contain the current user
+     * 
+     * The object \CActiveDataProvider returned is used to generate the view in application/views/admin/usergroup/usergroups_view.php
+     * 
+     * @param bool $mine
+     * @return \CActiveDataProvider
+     */
     function searchMine($mine)
```
Comment (olleharstedt) [inline application/models/UserGroup.php]: Thanks for your update. I'm going to be a bit petty and ask you to rename your `$mine` variable. Here are a couple of examples of how a boolean variable could be named to make clear its function:

```

 isFish = true
 isCat = false
 hasScales = true
 hasFur = false
 canSwim = true
 wasEgg = true
 eatAble = true
```

Google for more info about best practice.

**Comment by papa-mb** [comment]
Hi this is done.
Regards

**Comment by olleharstedt** [comment]
Thanks.

**Pull request merged**